### PR TITLE
bazel/README.md: comment LLVM version

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Long Dai <long0dai@foxmail.com>

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/main/PULL_REQUESTS.md)

Commit Message:

There is not clear to comment LLVM version, so add it.

Additional Description:
Risk Level: LOW
Testing:
Docs Changes: Y
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Comment by moderation** [comment]
Wondering if it is time to bump to 11.1.0?
```
rg -e '11\.0\.1'
tools/api_boost/api_boost.py                                                                                                               
148:        'clang/11.0.1/include')                                                                                                        
                                                                                                                                           
tools/code_format/check_format.py                                                                                                          
360:                "    export CLANG_FORMAT=clang-format-11.0.1\n"                                                                        
                                                                                                                                           
test/run_envoy_bazel_coverage.sh                                                                                                           
5:LLVM_VERSION="11.0.1"                                                                                                                    
                                                                                                                                           
bazel/repository_locations.bzl                                                                                                             
867:        version = "11.0.1",                                                                                                            
                                                                                                                                           
bazel/README.md                                                                                                                            
580:curl -sSfL "https://github.com/llvm/llvm-project/archive/llvmorg-11.0.1.tar.gz" | tar zx                                               
591:  -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX="/opt/libcxx_tsan" "../llvm-project-llvmorg-11.0.1/llvm"
885:export CLANG_FORMAT="$HOME/ext/clang+llvm-11.0.1-x86_64-linux-gnu-ubuntu-20.04/bin/clang-format"                                       
```

**Comment by daixiang0** [comment]
`x.y.0` is a minor upgrade without bugfix, so that `11.0.1` is more stable than `11.1.0` I think,

**Comment by daixiang0** [comment]
@phlax hi, seems `/retest` does not re-run test, since [it](https://dev.azure.com/cncf/envoy/_build/results?buildId=76008&view=logs&s=a34f18f2-f3fe-5fa3-4200-1597f6736c14&j=372032e6-8bf2-5d86-e328-540457529814) turns out timeout while I just call it in 5 minutes.

**Comment by daixiang0** [comment]
@phlax @lizan please review.

**Comment by phlax** [comment]
@daixiang0 my hesitancy here is whether - as suggested - we should first bump clang/llvm to 11.1.0 first

do you want to open a PR on envoy-build-tools to do that?

File: bazel/README.md [hunk 1]
```diff
@@ -85,6 +85,9 @@ for how to update or override dependencies.
 
     ### Linux
     On Linux, we recommend using the prebuilt Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html).
+
+    Now we use 11.0.1 version.
```
Comment (wrowe) [inline bazel/README.md:89]: I'd simplify... and concatenate this to the first statement... "from [LLVM...](...), version 11.0.1 or higher."

This change goes out of date immediately when we bump the linux toolchain. On the windows toolchain, we are at 11.1.0.

**Review by wrowe — CHANGES_REQUESTED** [review]

File: bazel/README.md (hunk 1 above)
Comment (lizan) [reply bazel/README.md:89]: We don't require 11.0.1, I do still use LLVM 9 locally sometime.

Comment (daixiang0) [reply bazel/README.md:89]: ping @lizan

**Comment by daixiang0** [comment]
@lizan could help advise?

**Comment by lizan** [comment]
I would add a link to https://github.com/envoyproxy/envoy/blob/main/ci/README.md where we have current clang version noted.

**Comment by daixiang0** [comment]
@lizan please review it.

File: ci/README.md [hunk 2]
```diff
@@ -48,6 +48,8 @@ As of November 2019 after [#8859](https://github.com/envoyproxy/envoy/pull/8859)
 To override the C++ standard library in your build, set environment variable `ENVOY_STDLIB` to `libstdc++` or `libc++` and
 run `./ci/do_ci.sh` as described below.
 
+Support Clang+LLVM package from [LLVM official site](http://releases.llvm.org/download.html) from 11.0.1 or higher.
```
Comment (wrowe) [inline ci/README.md:51]: I concur with Lizan, if you look two sections higher, under "Build image base and compiler versions" we explain this. It doesn't need to be called out under thit C++ standard library section. (That section too is out of date.) I think the change is superfluous, there is already too much overlapping documentation of the recommended tools.

See https://www.envoyproxy.io/docs/envoy/latest/start/building#linux-mac-target-requirements for the official location of such details.

**Review by wrowe — CHANGES_REQUESTED** [review]

File: ci/README.md (hunk 2 above)
Comment (daixiang0) [reply ci/README.md:51]: Oh, seems we'd better remove redundant contents? @wrowe

Comment (wrowe) [reply ci/README.md:51]: Yes, I agree we should streamline the prerequisites for the published docs and internally within the source tree. That seems like a topic for a distinct PR, help is welcome!

**Comment by wrowe** [comment]
I don't see any remaining change on this PR that clarifies the docs. If you would like to help focus the information in the appropriate sections, a new PR would be welcome. Closing this PR as asked and resolved.

**Pull request closed without merging**